lib/lightning_web/components/new_inputs.ex

```diff
@@ -338,7 +337,7 @@ defmodule LightningWeb.Components.NewInputs do
     ~H"""
     <p
       data-tag="error_message"
-      class="mt-3 inline-flex items-center gap-x-1.5 text-xs text-danger-600 phx-no-feedback:hidden"
```

Comment:
@jyeshe I am wondering if this related to what I am describing here

https://www.loom.com/share/31b4520022034160bee2e7b3f25b5ee2?sid=0b78c88c-e622-4e86-8c95-dc5b474cfc39

Reply:
Hey @elias-ba this nice improvement. I think we have it now in a generalized way and  customizable for other forms.

Reply:
Hey @jyeshe I just tested this again and it feels like we are no longer able to produce errors. I am wondering if this is what we want or I am missing something. See loom: https://www.loom.com/share/03543485934046bd8cc8d76d47390b0e?sid=b39eedfa-5212-4a44-81c9-cc310242986f

Reply:
It would show now for fields that were edited and deleted. For fields that were not edited yet it applies the suggestion from previous loom.